woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -206,6 +207,54 @@ def test_accessor_getattr(sample_df):
         sample_df.ww.not_present
 
 
+def test_getitem(sample_df):
+    df = sample_df.copy()
```

Comment:
I don't think we need these copy statements in the new tests, since we aren't modifying `sample_df` during the test.  If you want to leave df = sample_df, that's fine as it makes the statements that follow a little cleaner, or you could remove this completely and just use `sample_df` in place of `df`.

Reply:
✅  470d855